packages/client/components/ReflectionGroup/RemoteReflection.tsx

```diff
@@ -49,8 +45,10 @@ const RemoteReflectionModal = styled('div')<{
     isDropping ? Times.REFLECTION_REMOTE_DROP_DURATION : Times.REFLECTION_DROP_DURATION
   }ms ${BezierCurve.DECELERATE}`,
   transform,
-  animation:
-    isSpotlight && !isDropping ? `${circleAnimation(transform)} 3s ease infinite;` : undefined,
+  transformOrigin: '0 0',
+  offsetPath: triquetaPath,
+  offsetRotate: '0deg',
+  animation: isSpotlight && !isDropping ? `${spotlightWiggle} 4s linear infinite;` : undefined,
```

Comment:
Now that we're not using `transform` here, it looks like we can remove the `transform` on line 47, remove `isSpotlight` from `getStyle`, and remove the first if statement in `getStyle` as well as the comment above the `getStyle` function

Reply:
Not using `transform` is the reason the animation is offset, isn't it?

Reply:
Yes, the animation is using a path, but the transform property of the previous animation was added for an offset. I think it was dynamic or passed from the parent component or something similar, that's why I added it there. Sadly I didn't leave a comment.